core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigrationActor.scala

```diff
@@ -572,10 +573,11 @@ object WorkspaceMigrationActor {
           serviceAccount <- storageTransferService.getStsServiceAccount(googleProject)
           serviceAccountList = NonEmptyList.one(Identity.serviceAccount(serviceAccount.email.value))
 
-          // STS requires the following to read from the origin bucket
+          // STS requires the following to read from the origin bucket and delete objects after
+          // transfer
           _ <- storageService.setIamPolicy(originBucket, Map(
             StorageRole.LegacyBucketReader -> serviceAccountList,
-            StorageRole.ObjectViewer -> serviceAccountList
+            StorageRole.ObjectAdmin -> serviceAccountList
```

Comment:
https://cloud.google.com/storage-transfer/docs/configure-access says LegacyBucketWriter and ObjectViewer are required

Reply:
To delete objects from the source, it needs to be more than an object viewer?

Reply:
Thanks - I should have double checked that.

Reply:
Can you thunb this too https://github.com/broadinstitute/workbench-libs/pull/996 please?